cocos/core/memop/cached-array.ts

```diff
@@ -80,20 +85,22 @@ export class CachedArray<T> extends ScalableContainer {
      * @en
      * Pop the last element in the array. The [[length]] will reduce, but the internal array will keep its size.
      * @zh
-     * 弹出数组最后一个元素，CachedArray 的 [[length]] 会减少，但是内部数组的实际长度不变
-     * @return The last element.
+     * 弹出数组最后一个元素，CachedArray 的 [[length]] 会减少，但是内部数组的实际长度不变。
+     * @return @en The last element of this CachedArray. If CachedArray is empty, will return undefined.
+     * @zh 数组的最后一个元素。如果数组为空，将返回 undefined。
      */
     public pop (): T | undefined {
         return this.array[--this.length];
     }
 
     /**
      * @en
-     * Get the element at the specified index of the array
+     * Get the element at the specified index of the array.
      * @zh
-     * 得到数组中指定位置的元素
-     * @param idx The index of the requested element
-     * @return The element at given index
```

Comment:
`return` or `returns`?

Reply:
If you use VSCode to add comments, it will show you `returns`.

Reply:
There are still a lot of `return`.
![image](https://user-images.githubusercontent.com/15109777/209086813-42b91041-c360-44bd-9d85-27434a1a3bcd.png)

Reply:
fixed

Reply:
The API doc tool syntax is @return though, this is not related to grammar check